Sources/LiveViewNative/Views/PhxHeroView.swift

```diff
@@ -1,166 +1,166 @@
+////
+////  PhxHeroView.swift
+////  LiveViewNative
+////
+////  Created by Shadowfacts on 5/2/22.
+////
 //
-//  PhxHeroView.swift
-//  LiveViewNative
-//
-//  Created by Shadowfacts on 5/2/22.
-//
-
-import SwiftUI
-
-struct PhxHeroView<R: CustomRegistry>: View {
```

Comment:
@shadowfacts Is `PhxHeroView` something that should be in this package? I can't tell what SwiftUI functionality it's wrapping.

Reply:
No, it shouldn't be anymore. Originally it was part of the hero navigation transition, but that's been non-functional since iOS 16. If it returns, it'll either be as a wrapper for `.matchedGeometryEffect` (if that eventually supports nav transitions) or as a separate package if we're reimplementing it ourselves.

Reply:
Ok, I'll remove it completely.